exec_syscall.c

```diff
@@ -0,0 +1,13 @@
+#include <common.h>
+#include <exec.h>
+
+int main() {
```

Comment:
Well, I've tried to figure it out but with no luck - I found out that "The semihost libraries for some processors do not support this feature" in [OVPsim user guide](http://www.ovpworld.org/documents/Simulation_Control_of_Platforms_and_Modules_User_Guide.pdf) (page 30) and nothing really helpful on the internet.